services/net/syndication/SyndicationAction.cs

```diff
@@ -259,6 +260,37 @@ private SourceContent CreateSourceContent(IngestModel ingest, SyndicationItem it
         };
     }
 
+    /// <summary>
+    /// swap line breaks with paragraph tags
+    /// </summary>
+    /// <param name="text"></param>
+    /// <returns>formatted body string</returns>
+    protected string SanitizeLineBreaks(string? text)
```

Comment:
Can we move this to a shared library so that we have consistent implementation on different Ingests?